### PR TITLE
Update uuid to v8.0.0

### LINKED ISSUE #66: Update uuid dependency to ^8.0.0
# Feature Request

## Description

Update the [uuid dependency](https://github.com/cerner/terra-application/blob/99204aafa7ba001f531e4e219d597b93666eb359/package.json#L122) from ^3.0.0 to ^8.0.0.

## Additional Context / Screenshots

Installing uuid in a project that contains terra-application will output warnings for duplicate versions of uuid.

https://github.com/uuidjs/uuid

```
WARNING in uuid
  Multiple versions of uuid found:
    3.4.0 ./~/terra-application/~/uuid
    8.3.0 ./~/uuid
```

### PR DESCRIPTION
**Pull request by saket2403**
### Summary

Updated the uuid dependency from ^3.0.0 to ^8.0.0.

Closes #66 

### Deployment Link

https://terra-applic-.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
     "terra-status-view": "^4.10.0",
     "terra-theme-context": "^1.0.0",
     "terra-theme-provider": "^4.0.0",
-    "uuid": "^3.0.0",
+    "uuid": "^8.0.0",
```
Comment (StephenEsser) [inline package.json:123]: There were non-passive import changes from v3 -> v8. The import declaration will need to change in the files consuming uuid.

https://github.com/uuidjs/uuid#quickstart

```
import { v4 as uuidv4 } from 'uuid';
```

**Comment by mjhenkes** [comment]
Please re-open this PR once you have a working build.

File: package.json (hunk 1 above)
Comment (saket2403) [reply package.json:123]: Done

**Comment by saket2403** [comment]
> Please re-open this PR once you have a working build.

npm install was failing for some reason, working now!

**Review by StephenEsser — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**